packages/contracts-bedrock/src/legacy/DeployerWhitelist.sol

```diff
@@ -3,8 +3,8 @@ pragma solidity 0.8.15;
 
 import { ISemver } from "src/universal/ISemver.sol";
 
-/// @custom:legacy
-/// @custom:proxied
+/// @custom:legacy true
```

Comment:
Do we even still need this contract in the repo?

Reply:
Mmm let's open a separate issue for this

Reply:
Opened #11767